src/Workspaces/Core/Desktop/PublicAPI.Shipped.txt

```diff
@@ -1,44 +1,11 @@
 Microsoft.CodeAnalysis.CommandLineProject
-Microsoft.CodeAnalysis.FileTextLoader
-Microsoft.CodeAnalysis.FileTextLoader.DefaultEncoding.get -> System.Text.Encoding
-Microsoft.CodeAnalysis.FileTextLoader.FileTextLoader(string path, System.Text.Encoding defaultEncoding) -> void
-Microsoft.CodeAnalysis.FileTextLoader.Path.get -> string
```

Comment:
You're totally right. Will do. Greath catch!

Reply:
I've pushed a type forwarder that is `#if false'd` for now since the public API analyzer in the roslyn rep doesn't have the support for type forwarders yet. Once https://github.com/dotnet/roslyn/pull/25179 is merged, I should be able to enable the type forwarder and update the public API files.